### DISCUSSION TITLE
Suggestion: Mod Compatability wtih Alex's Mobs

### DISCUSSION THREAD

**Discussion by Daralexen05** [Faboslav/friends-and-foes#216] [Ideas]
interactions between mobs from this mod and Alex's Mobs.
1. Maulers attack Road Runners, Raccoons, Lobsters, Crows, Capuchin Monkeys, Jerboas, Rain Frogs, Baby Emus, and Baby Kangaroos
2. Mooblooms can be pollinated by Hummingbirds just like they can by Bees
3. An interaction between Rascal and Underminer since both are inhabitants of the Abandoned Mineshaft structure
4. An interaction between Glare and Potoo since both share the same function of warning the player of dark areas
5. Iceologers attack or scare off Froststalkers or Snow Leopards that attack the Pigs or Goats that spawn in their Cabins
6. Possible interaction between Barnicle and aquatic life of Alex's Mobs

**Comment by Faboslav** [comment]
Hello, i am sorry, but i dont have time to do the compat implementations like these, also there are certain personal reasons, why i dont want to do compat with alex mobs :).

**Reply by Daralexen05** [reply]
What are the personal reasons? Do you disagree with the design choices in Alex's Mobs? I love the mod, but there are some glaring examples of bad game design and power creep.
